RFC.rst

```diff
@@ -49,7 +50,6 @@ Examples of unacceptable behaviour by participants include:
   * Trolling or insulting/derogatory comments
   * Public or private harassment
   * Publishing other's private information, such as physical or electronic addresses, without explicit permission
-  * Other unethical or unprofessional conduct
```

Comment:
This makes sense. It's better to not make it too broad an ambiguous, but add additional clauses as the specific issues arise.